clang/test/Driver/cheri/freebsd-cheri-libcheri.c

```diff
@@ -0,0 +1,41 @@
+// RUN: %plain_clang_cheri_triple_allowed -no-canonical-prefixes \
```

Comment:
This substitution should probably be removed, I originally added it when there were weird crashes if you didn't pass -fPIC to clang when compiling CHERI code.